### PR TITLE
Added few tests and made some minor changes in the code. 

### PR DESCRIPTION
**Pull request by sanyamsinghal**
The tests are added under the `test` directory of the repo.
the `package.json` file has also been updated to run the test using `npm test` command.

### CODE REVIEW

**Review by ashetkar — COMMENTED** [review]
Please add more details on how to run the tests with pre-requisites so that it'll be easier for the users.
I tried to run `npm test` on the master but it failed saying `mocha: command not found`. So installed mocha.

But I still see error like `SequelizeConnectionError: database "test_sequelize" does not exist`.
I have a YBDB cluster running locally on mac. Does one need a Docker container? If yes, please add so in the pre-requisites.

**Comment by sanyamsinghal** [comment]
Hi, @ashetkar I have updated the README file for How to run the tests. Please have a look and let me know.
No, you don't need a docker container, having a local cluster on your mac will work.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,24 @@ Use the below command to install [sequelize-yugabytedb](https://www.npmjs.com/pa
 npm install sequelize-yugabytedb 
 ```
 
+### Steps to run the tests
+1. Make sure you have a YugabyteDB cluster up and running on your machine. <br/>Note: The tests are assuming the `localhost` and `5433` as default host and port for YugabyteDB cluster.
+2. Create a database `test_sequelize`:
+   ```
+   $ ysqlsh -c "CREATE DATABASE test_sequelize"
```
Comment (ashetkar) [inline README.md]: Can be more explicit:
Move to YugabuteDB installation directory and run ```./bin/ysqlsh ...```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,24 @@ Use the below command to install [sequelize-yugabytedb](https://www.npmjs.com/pa
...
+   ```
+3. Install dependencies by running:
```
Comment (ashetkar) [inline README.md]: Similarly:
Move back to `sequelize-yugabytedb` directory and ...

**Review by ashetkar — APPROVED** [review]
LGTM with minor suggestions.

**Pull request merged**